packages/thirdweb/src/exports/wallets.ts

```diff
@@ -119,3 +119,5 @@ export type {
 
 // injected
 export { injectedProvider } from "../wallets/injected/mipdStore.js";
+
+export { getValidChainRPCs } from "../wallets/utils/chains.js";
```

Comment:
this is very wallet internal, I know there's a place dashboard uses it but in there can we instead... just not use it and instead always use the thirdweb RPC?